examples/multibody/deformable_torus/deformable_torus.cc

```diff
@@ -37,106 +42,228 @@ using drake::math::RigidTransformd;
 using drake::multibody::AddMultibodyPlant;
 using drake::multibody::Body;
 using drake::multibody::CoulombFriction;
+using drake::multibody::DeformableBodyId;
+using drake::multibody::DeformableModel;
+using drake::multibody::ExternalForceField;
+using drake::multibody::ModelInstanceIndex;
+using drake::multibody::MultibodyPlant;
 using drake::multibody::MultibodyPlantConfig;
 using drake::multibody::Parser;
 using drake::multibody::PrismaticJoint;
+using drake::multibody::SpatialInertia;
 using drake::multibody::fem::DeformableBodyConfig;
-using drake::multibody::DeformableBodyId;
-using drake::multibody::DeformableModel;
+using drake::multibody::internal::MultibodyTreeSystem;
 using drake::systems::BasicVector;
 using drake::systems::Context;
 using Eigen::Vector2d;
+using Eigen::Vector3d;
 using Eigen::Vector4d;
 using Eigen::VectorXd;
 
 namespace drake {
 namespace examples {
 namespace {
 
-/* We create a leaf system that uses PD control to output a force signal to
- a gripper to follow a close-lift-open motion sequence. The signal is
- 2-dimensional with the first element corresponding to the wrist degree of
- freedom and the second element corresponding to the left finger degree of
- freedom. This control is a time-based state machine, where forces change based
- on the context time. This is strictly for demo purposes and is not intended to
- generalize to other cases. There are four states: 0. The fingers are open in
- the initial state.
-  1. The fingers are closed to secure a grasp.
-  2. The gripper is lifted to a prescribed final height.
-  3. The fingers are open to loosen a grasp.
- The desired state is interpolated between these states. */
-class GripperPositionControl : public systems::LeafSystem<double> {
+class PointSourceForceField : public ExternalForceField<double> {
```

Comment:
Is the `PointSourceForceField` just for the sake of illustration for how a user might create their own `ExternalForceField`, or is it an actual class you plan to implement in Drake?  (if the latter, I have a few comments)